File: 1480-72-1790-check-if-one-string-swap-can-make-strings-equal/1480-72-1790-check-if-one-string-swap-can-make-strings-equal.py
class Solution:
    def areAlmostEqual(self, s1: str, s2: str) -> bool:
        diff = [(i, j) for i, j in zip(s1, s2) if i != j]
        return len(diff) == 0 or (len(diff) == 2 and diff[0][::-1] == diff[1])

File: 1480-72-1790-check-if-one-string-swap-can-make-strings-equal/1480-72-1790-check-if-one-string-swap-can-make-strings-equal.js
var areAlmostEqual = function(s1, s2) {
    let diff = [];
    
    for (let i = 0; i < s1.length; i++) {
        if (s1[i] !== s2[i]) diff.push([s1[i], s2[i]]);
    }
    
    return diff.length === 0 || (diff.length === 2 && diff[0].reverse().toString() === diff[1].toString());
};
